packages/package.go

```diff
@@ -216,9 +216,44 @@ func getDownloadPath(p Package, t string) string {
 	return path.Join("/epr", p.Name, p.Name+"-"+p.Version+".zip")
 }
 
+// MustParsePackage creates a new package instances based on the given base path.
+// The path passed goes to the root of the package where the manifest.yml is.
+// It runs more strict validation than NewPackage, e.g. ensuring that all categories are valid.
+func MustParsePackage(logger *zap.Logger, basePath string, fsBuilder FileSystemBuilder) (*Package, error) {
+	p, err := newPackage(logger, basePath, fsBuilder)
+	if err != nil {
+		return nil, fmt.Errorf("failed to create package from path %s: %w", basePath, err)
+	}
+	for _, c := range p.Categories {
+		if _, ok := Categories[c]; !ok {
+			return nil, fmt.Errorf("invalid category: %s", c)
```

Comment:
Should we show the name and version in the message?
```suggestion
			return nil, fmt.Errorf("invalid category in package %s-%s: %s", p.Name, p.Version, c)
```